docs/development.md

```diff
@@ -45,7 +45,7 @@ Click the "Terminate" icon on the Eclipse console.
 1. Click `+ → Google AppEngine Dev Server`.
 1. Name it `Dev Server`.
 1. Click `Configure` next to `Application server`.
-1. Click `+ → ...`. Select the App Engine SDK you downloaded in Step 3 of the [Setting up a development environment](settingUp.md) guide.
+1. Click `+ → ...`. Select the App Engine SDK (appengine-java-sdk-<version> sub-folder) you downloaded in Step 3 of the [Setting up a development environment](settingUp.md) guide.
```

Comment:
So 
it should look like something this way
`Click `+ → ...`. Select the App Engine SDK→ appengine-java-sdk-<version> sub-folder you downloaded in Step 3 of the [Setting up a development environment](settingUp.md) guide.`